part-overview.tex

```diff
@@ -71,7 +71,7 @@ \section{Application architecture}
 \href{https://webpack.js.org/}{webpack 5} and \href{https://developer.mozilla.org/en-US/docs/Web/Web_Components}{Web Components}) and old (e.g. \href{https://dojotoolkit.org/}{Dojo Toolkit}) technology.
 
 The web server is an optional component that's highly recommended for TLS termination
-and protecting the back end Perl HTTP server from malicious traffic. It may also be used
+and protecting the back end Perl HTTP server from malicious traffic\footnote{This setup is an industry-wide accepted best practice for deploying web applications; web servers like Apache or Nginx have been and continue to extensively security-reviewed. No other implementations can achieve the same level of scrutiny.}. It may also be used

```

Comment:
"to extensively security-reviewed" should be "to be extensively security-reviewed"